language-support/hs/bindings/src/DA/Ledger/Services/CommandCompletionService.hs

```diff
@@ -1,28 +1,26 @@
 -- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 -- SPDX-License-Identifier: Apache-2.0
 
-module DA.Ledger.Services.CommandCompletionService (completionStream) where
+{-# LANGUAGE GADTs #-}
```

Comment:
Is this required?

Reply:
Yes. The `ClientResult` type form the grpc library is a GADT. This is patterns matched in this file at line 60.